modules/RaptMedia/resources/mw.RaptMedia.js

```diff
@@ -157,8 +156,12 @@
 		},
 
 		readRaptProjectId: function() {
-			var partnerData = this.getPlayer().evaluate('{mediaProxy.entry.partnerData}');
-			var segments = (partnerData || "").split(';');
+			var partnerData = this.getPlayer().evaluate('{mediaProxy.entry.partnerData}') || '';
+			if (partnerData.toLowerCase() === 'raptmedia!') {
```

Comment:
is this used for some kind of distinction between old and new playlists?

Reply:
Yes. For existing (old) rapt backend based projects this field contains `raptmedia;[RAPT PROJECT ID]`.

This adds a check for `raptmedia!` which it treats as a (new) kaltura based project.

Is there a better field to use for this?